sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2056,11 +2049,11 @@
       "properties": {
         "primaryKey": {
           "type": "string",
-          "description" : "The base 64 encoded primary key of your device."
+          "description": "The base 64 encoded primary key of your device."
```

Comment:
the device?

Reply:
Not sure who is doing the 'device' endpoint, but this change is to remove an extra space. I suggest we make the change you suggested in a different PR that does all of the device paths